Datepicker.js

```diff
@@ -147,6 +147,22 @@ class clsDatepicker {
             this.containerElement.querySelector('.startDateElement').innerHTML = "Date: " + this.dates[0];
         }
         this.highlightDates();
+		
+		//autoClose the claendar when a single date or date range is selected 
+		if(!this.singleDate && this.dates.length === 2 && this.options.autoClose)
+		{
+			setTimeout(function()
+			{ 
+				this.containerElement.hideEl();
+			}.bind(this), 400);
+		}
+		else if(this.singleDate && this.dates.length === 1 && this.options.autoClose)
+		{
+			setTimeout(function()
+			{ 
+				this.containerElement.hideEl();
+			}.bind(this), 400);
+		}
```

Comment:
back up indentation to match the rest of the code formatting

Reply:
If your version of VSCode won't format this properly I can still merge and fix it later on the master branch.

Reply:
Not even Github's editor formats like this. You know my feelings about it but this is your project.